tests/regression/src/lib.rs

```diff
@@ -303,7 +322,7 @@ mod tests {
     /// Test to create new config, set security policy, host_callback information, load/trust certs, and build config.
     #[test]
     fn test_set_config() {
-        valgrind_test("test_set_config", |ctrl| {
+        valgrind_test("test_set_config", 0.01, |ctrl| {
```

Comment:
Where did 0.01 come from? Did you try this with any PRs that are currently out right now/recent PRs that might have changed the instruction counts?

Reply:
That .01 comes from running that test 10 times for the same PR and noting the amount of non-determinism (performance difference in identical versions of s2n-tls). Since it averages around .005 for my tests, I decided that .01 is a reasonable threshold to indicate a significant performance difference that is not attributed to non-determinism.

Reply:
Yeah, the reason why I suggested running it with different PRs that are out right now is that we don't want to merge this and instantly block the s2n-tls CI. But it seems like you've tested enough to ensure that we won't.